.travis.yml

```diff
@@ -0,0 +1,35 @@
+language: java
+jdk:
+  - oraclejdk11
+
+script:
+  - cd complete
+  - mvn package
+  - zip -r latest *
```

Comment:
I've always wondered, is it just a standard to zip all the files? Can't we just take the jar?